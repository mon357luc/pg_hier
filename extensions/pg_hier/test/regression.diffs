diff -U3 /extensions_src/pg_hier/test/expected/setup_taxon_schema.out /extensions_src/pg_hier/test/results/setup_taxon_schema.out
--- /extensions_src/pg_hier/test/expected/setup_taxon_schema.out	2025-07-15 02:48:15.790734900 +0000
+++ /extensions_src/pg_hier/test/results/setup_taxon_schema.out	2025-07-15 03:17:30.997837800 +0000
@@ -286,4 +286,4 @@
 ('Pinus pinaster', 'Maritime pine', 17, 'Pine native to the Mediterranean'),
 ('Pinus pinea', 'Stone pine', 17, 'Pine that produces edible pine nuts'),
 ('Amanita verna', 'Fool''s mushroom', 18, 'Deadly poisonous white mushroom'),
-('Amanita citrina', 'False death cap', 18, 'Yellow-tinged mushroom');
+('Amanita citrina', 'False death cap', 18, 'Yellow-tinged mushroom');
