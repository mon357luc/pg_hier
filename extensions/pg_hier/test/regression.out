# using postmaster on Unix socket, default port
not ok 1     - setup_taxon_schema                        337 ms
# diff command failed with status 512: diff  "/extensions_src/pg_hier/test/expected/pg_hier_edge_cases_test.out" "/extensions_src/pg_hier/test/results/pg_hier_edge_cases_test.out" > "/extensions_src/pg_hier/test/results/pg_hier_edge_cases_test.out.diff"Bail out!
